tox.ini

```diff
@@ -3,6 +3,7 @@ minversion = 3.5.0
 skipdist = True
 skip_missing_interpreters = True
 isolated_build = True
+toxworkdir = {env:TOXWORKDIR:/tmp/{env:USER}/tox}
```

Comment:
Shouldn't the work dir be a tempdir, uniquely named per run?

Reply:
I presume this means: get the value of TOXWORKDIR from the environment and if that is not defined use /tmp/$USER/tox - correct?

Reply:
> Shouldn't the work dir be a tempdir, uniquely named per run?

It doesn't need to be.  Tox takes are of the per-run stuff.  Without this change, we use the source tree where the `tox.ini` is located as the "work" directory [1].

[1] https://tox.readthedocs.io/en/latest/config.html#conf-toxworkdir